src/assets/sass/protocol/components/_call-out.scss

```diff
@@ -0,0 +1,219 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../includes/lib';
+
+/* -------------------------------------------------------------------------- */
+// Call Out component
+
+.mzp-c-call-out {
+    background-color: $color-gray-20;
+    text-align: center;
+
+    &.mzp-t-dark {
+        background-color: $color-gray-90;
+        color: $color-white;
+
+        .mzp-c-call-out-desc {
+            color: $color-gray-30;
+        }
+    }
+
+    .mzp-c-call-out-title {
+        @include text-display-md;
+    }
+
+    .mzp-c-call-out-desc {
+        @include text-body-lg;
+        margin-bottom: 0;
+    }
+
+    .mzp-c-button-download {
+        margin: $margin-lg 0 0;
+    }
+
+    &.mzp-t-product-firefox ,
+    &.mzp-t-product-beta,
+    &.mzp-t-product-developer,
+    &.mzp-t-product-nightly,
+    &.mzp-t-product-focus {
```

Comment:
Instead of writing out these class names it might make more sense to have one theme `.mzp-t-product-firefox` that acts as a generic firefox theme and if and if you wanted to theme something to a sub product, Firefox Focus for instance, we would add both the `.mzp-t-product-firefox` class and the `.focus` class to the component.

Reply:
Good call 👍

Reply:
On second thought, I think this maybe requires a little more consideration. The one snag here is introducing a non-mzp prefixed class name, which is something we should discourage in general to avoid conflicts in third-party CSS. Adding too many `mzp` class names to a single component for theming also gets a little verbose.

In the future we may also want to add more products to this that don't fall into the Firefox line of products, so perhaps this is the more future proof method for now.